doc/windows/package-manager/package/manifest.md

```diff
@@ -18,7 +16,6 @@ This article describes the contents of a package manifest for Windows Package Ma
 
 The YAML format was chosen for package manifests because of its relative ease of human readability and consistency with other Microsoft development tools. If you are not familiar with YAML syntax, you can learn the basics at [Learn YAML in Y Minutes](https://learnxinyminutes.com/docs/yaml/).
```

Comment:
I feel like there should be some mention of [microsoft/winget-create](https://github.com/microsoft/winget-create) here. If someone doesn't know YAML, that might be an easier way for them to get started.

(You don't have to do it @vedantmgoyal2009, just something I noticed looking over this).

Reply:
I have added a section for using `wingetcreate` and the YamlCreate script. Thanks for the suggestion.